models/dps/params.go

```diff
@@ -24,6 +24,10 @@ const (
 	FlowTestnet    = flow.Testnet
 	FlowSymbol     = "FLOW"
 	FlowDecimals   = 8
+
+	StatusCompleted = "COMPLETED"
```

Comment:
Related to my comment on the other PR - I saw that in the `/network/options` endpoint they in fact use uppercase for some of the fields.

In the `/block/transaction` endpoint they use only the first letter uppercased - https://www.rosetta-api.org/docs/models/Transaction.html

Not sure what are their rules about this stuff.

Reply:
That's not part of the specification. Possible status and type of operations are defined by the implementation.